### ISSUE TITLE
rclone warns about control characters and non file/directory (anything not Storable) for filtered files

### ISSUE THREAD

**Issue by jikamens** [rclone/rclone#1646]
If there's a file in the source that isn't storable (i.e., Storable in local/local.go returns false), then rclone warns about it at level NOTICE even when the filters specified by the user would exclude it.

If a file is filtered out, it shouldn't be warned about. If it's filtered out, it's not a problem that needs to be warned about. ;-)

**Comment by ncw** [comment]
Yes this is a known problem.  The logging happens at a layer below the filtering so it logs stuff even for filtered files :-(

Not sure what to do about this!

**Comment by ivandeex** [comment]
@jikamens Does the problem reproduce with rclone 1.54?

**Comment by jikamens** [comment]
Still broken in 1.54.

**Comment by roucc** [comment]
This has been fixed by running the filters in the local backend.
